src/AppInstallerCLITests/Dependencies.cpp

```diff
@@ -206,4 +208,69 @@ TEST_CASE("DependencyNodeProcessor_NoMatches", "[dependencies]")
     REQUIRE(dependencyList.Size() == 0);
     REQUIRE(installOutput.str().find(Resource::LocString(Resource::String::DependenciesFlowNoMatches)) != std::string::npos);
     REQUIRE(result == DependencyNodeProcessorResult::Error);
-}
\ No newline at end of file
+}
+
+std::string GetCurrentProcessPathVariable()
+{
+    size_t requiredSize;
+    getenv_s(&requiredSize, nullptr, 0, "PATH");
+
+    if (requiredSize > 0)
+    {
+        char* buffer = new char[requiredSize];
+        errno_t errorResult = getenv_s(&requiredSize, buffer, requiredSize, "PATH");
+        if (errorResult == 0)
+        {
+            return std::string(buffer);
+        }
+
+        delete[] buffer;
+    }
+    return {};
+}
+
+TEST_CASE("RefreshEnvironmentVariable_User", "[dependencies]")
+{
+    std::string testPathEntry = "testUserPathEntry";
```

Comment:
Since we are running all these tests on our dev machines, I don't think we would want these tests mess with our registry directly.

For now, maybe just guard it behind admin check.

Reply:
Added admin check to unit test.